healpy/src/_sphtools.pyx

```diff
@@ -574,6 +574,7 @@ def rotate_alm(alm not None, double psi=0, double theta=0, double phi=0, matrix=
         del AI, AG, AC
 
 
+@cython.cdivision(True)
```

Comment:
why do you add this?

Reply:
I was getting a compiler error with Cython 3.0, which I think is related to the default language level change in Cython 3.0.